### PR TITLE
Kindsys: Rename mudball to common schemas, allow multiple input cue files

### PR DESCRIPTION
**Pull request by sdboyer**
**What is this feature?**

This PR does two things:
* Names (renames?) "the mudball" to "common schemas", and moves them from `packages/grafana-schema/src/schema` to `packages/grafana-schema/src/common`
* Allows the declarations to be made in multiple cue files (all having `package common`) to make it easier to organize it all more tidily.

Note that until we have grafana/cuetsy#53, which will take some work, it gets a little bit more awkward to declare common types in order, because it's not just order within one file, but order of file visits (lexicographic) and _then_ order within those files.

Not a big enough issue to be prohibitive, though, and not one with a subtle failure, so i think we're OK leaving that as a TODO.

**Why do we need this feature?**

It's messy keeping everything in a single file. A single generated output file is OK, though. 

And, "mudball" was always the word we used because we didn't have a better one. I think "common" does the job well, here.

**Who is this feature for?**

Anyone who has to write these common schemas should be more able to satisfy the programmer's 

**Which issue(s) does this PR fix?**:

related to #58711 

fyi @torkelo @ryantxu @joshhunt

### CODE REVIEW

File: packages/grafana-schema/tsconfig.build.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   // dashboard_experimental.gen.ts needs ignoring as isolatedModules requires it to contain an import or export statement.
-  "exclude": ["dist/**/*", "src/schema/dashboard/dashboard_experimental.gen.ts", "**/*.test.ts*"],
+  "exclude": ["dist/**/*", "**/*.test.ts*"],
```
Comment (Clarity-89) [inline packages/grafana-schema/tsconfig.build.json:3]: Probably should also remove the comment, it's not a valid json anyway.

File: packages/grafana-schema/src/common/mudball.cue [hunk 2]
```diff
@@ -203,7 +203,7 @@ FieldTextAlignment: "auto" | "left" | "right" | "center" @cuetsy(kind="type")
 
 // Internally, this is the "type" of cell that's being displayed
 // in the table such as colored text, JSON, gauge, etc.
-// The color-background-solid, gradient-gauge, and lcd-gauge 
+// The color-background-solid, gradient-gauge, and lcd-gauge
 // modes are deprecated in favor of new cell subOptions
 TableCellDisplayMode: "auto" | "color-text" | "color-background" | "color-background-solid" | "gradient-gauge" | "lcd-gauge" | "json-view" | "basic" | "image" | "gauge" @cuetsy(kind="enum",memberNames="Auto|ColorText|ColorBackground|ColorBackgroundSolid|GradientGauge|LcdGauge|JSONView|BasicGauge|Image|Gauge")
```
Comment (Clarity-89) [inline packages/grafana-schema/src/common/mudball.cue:208]: Maybe this should also be in the `table.cue`?

**Review by Clarity-89 — APPROVED** [review]
Looks good! Left a few minor comments.

File: packages/grafana-schema/src/common/mudball.cue (hunk 2 above)
Comment (sdboyer) [reply packages/grafana-schema/src/common/mudball.cue:208]: oh yeah probably

**Comment by Clarity-89** [comment]
9.4. hasn't been released yet, so that probably should be the milestone and not 9.5?

**Comment by sdboyer** [comment]
with milestones that aren't really focused on an actual public release narrative - which none of our schema stuff is until we formally go public with it next month - i just tend to pick a milestone that's further out so that there's little chance of release engineering tripping on it 🤷🏻

**Comment by Clarity-89** [comment]
Makes sense, but then we'd probably agree on some milestone since we have multiple PRs now depending on each other and those shouldn't have different milestones.

**Comment by sdboyer** [comment]
interesting - is that something that's actually tracked by our release process? i mean, that certainly makes sense in principle, and now i'm worried that i've perhaps been breaking something without realizing it 😱

**Comment by Clarity-89** [comment]
> interesting - is that something that's actually tracked by our release process? i mean, that certainly makes sense in principle, and now i'm worried that i've perhaps been breaking something without realizing it 😱

Good point, as far as I know, at least previously, every new minor release is branched off from main so technically this shouldn't break anything but might make it harder to track when changes were introduced, need to double check with the release engineering.

**Comment by sdboyer** [comment]
@malcolmholmes thoughts on this?

**Pull request merged**